packages/editor/src/core/components/Cell/index.tsx

```diff
@@ -107,39 +109,51 @@ const Cell: React.FC<Props> = ({ nodeId, measureRef }) => {
 
   return (
     <div
-      ref={ref}
+      style={{
+        padding: `0 ${cellSpacing / 2}px`,
+        paddingBottom:
+          (!hasChildren || hasPlugin) && !inline ? `${cellSpacing}px` : null,
+      }}
       className={classNames(
-        'react-page-cell',
-
         gridClass({
           isEditMode,
           isPreviewMode,
           size,
         }),
         {
           'react-page-cell-has-inline-neighbour': hasInlineNeighbour,
+          [`react-page-cell-inline-${inline || ''}`]: inline,
+        }
+      )}
```

Comment:
i seldom use inline cells, can't really tell if this change has any side effect.

Reply:
These classes declare various `float` styles and must be assigned to the top level div or the floats won't work. That's why these two classes were separated from the rest and moved to the outer div.